src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -20,83 +20,96 @@ all_data <- list.files(path = 'data/api/', full.names = TRUE) %>%
-                             ph_7 = ifelse(is.na(ph_7_nice), ph_7_other, ph_7_nice),
-
-                             #ORP
-                             orp_offset = ifelse(is.na(str_match(ph_orp, "Zobell'sOffset\\s*(.*?)\\s*mVTemperature")[,2]),
-                                                 str_match(ph_orp, "ZoBell'sOffset\\s*(.*?)\\s*mVTemperature")[,2],
-                                                 str_match(ph_orp, "Zobell'sOffset\\s*(.*?)\\s*mVTemperature")[,2]),
-
-                             #Conductivity
-                             tds_conversion_ppm = str_sub(str_match(conductivity, "TDSConversionFactor\\s*(.*?)\\s*CellConstant")[,2], 6, nchar(str_match(conductivity, "TDSConversionFactor\\s*(.*?)\\s*CellConstant")[,2])),
-                             cond_cell_constant = str_match(conductivity, "CellConstant\\s*(.*?)\\s*ReferenceTemperature")[,2],
-                             cond_pre = str_match(conductivity,paste0(str_match(conductivity,
-                                                                                "PreMeasurementActual\\s*(.*?)\\s*SpecificConductivity")[,2],"SpecificConductivity\\s*(.*?)\\s*µS/cmPost"))[,2],
-                             cond_post = str_match(conductivity,paste0(str_match(conductivity,
-                                                                                 "PostMeasurementActual\\s*(.*?)\\s*SpecificConductivity")[,2],"SpecificConductivity\\s*(.*?)\\s*µS/cm"))[,2],
-
-                             #Turbidity
-                             ntu_slope = str_match(turbidity, "Slope\\s*(.*?)\\s*Offset")[,2],
-                             ntu_offset = str_match(turbidity, "Offset\\s*(.*?)\\s*NTU")[,2],
-                             ntu_10 = str_match(turbidity, "CalibrationPoint1PreMeasurement\\s*(.*?)\\s*NTUPost")[,2],
-                             ntu_100 = str_match(turbidity, "CalibrationPoint2PreMeasurement\\s*(.*?)\\s*NTUPost")[,2],
-
-                             #Factory Defaults
-                             factory_defaults = paste0(ifelse(is.na(ntu_slope), "Turbidity ", ""),
-                                                       ifelse(is.na(rdo_slope), "RDO ", ""),
-                                                       ifelse(is.na(ph_slope_post), "pH ", ""),
-                                                       ifelse(is.na(orp_offset), "ORP ", ""),
-                                                       ifelse(is.na(cond_post), "Conductivity ", ""))) %>%
-      select(-c(ph_7_nice, ph_7_other))
-  }
-
-  cal_table <- bind_rows(cal_table) %>%
-    distinct(.keep_all = TRUE) %>%
-    group_by(site) %>%
-    mutate(DT = round_date(DT, "15 minutes")) %>%
-    padr::pad(by = 'DT', group = 'site')
+cal_files <- list.files("data/calibration_reports", pattern=".html")
+
+cal_tabler <- function(cal_files){
```

Comment:
consider using a different argument for the function here - like 'file' to eliminate confusion.